src/components/resource-title/ResourceTitle.scss

```diff
@@ -2,13 +2,14 @@
 
 .resource-title {
   align-items: center;
-  border-bottom: 1px solid var(--hauki-content-border-color);
-  box-shadow: 0 2px 10px 0 rgba(212, 212, 212, 0.2);
+  background: #fff;
```

Comment:
var(--color-white) perhaps?

Reply:
Yes. Need to start using those. I'll fix it.